apps/codeforafrica/src/components/ConnectWithMember/ConnectWithMember.js

```diff
@@ -0,0 +1,34 @@
+import { RichTypography } from "@commons-ui/core";
+import Stack from "@mui/material/Stack";
+import React from "react";
+
+import SocialMediaBar from "@/codeforafrica/components/SocialMediaBar";
+import SocialMediaConnectButton from "@/codeforafrica/components/SocialMediaConnectButton";
+
+const ConnectWithMember = React.forwardRef(function ConnectWithMember(
+  props,
+  ref
+) {
+  const { sx, title, links, ...other } = props;
+  return (
+    <Stack
+      direction="row"
+      {...other}
+      sx={{
+        color: "grey.main",
+        alignItems: "center",
+        ...sx,
+      }}
+      ref={ref}
+    >
+      <RichTypography variant="footerCap">{title}</RichTypography>
+      <SocialMediaBar links={links}>
+        {Object.entries(links).map(([key, value]) => {
+          return <SocialMediaConnectButton key={key} name={key} url={value} />;
+        })}
```

Comment:
We need to maintain the order of these social media:

![Screenshot from 2022-08-11 15-42-28](https://user-images.githubusercontent.com/1779590/184135909-dc85df95-285a-436a-9302-df68d05ca91e.png)